SDAPackage/SDA/obstacle_map.py

```diff
@@ -158,20 +159,24 @@ def generate_possible_paths(self, obstacle):
                     [obstacle.get_point()[0] + obstacle.get_radius(), obstacle.get_point()[1], obstacle.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)],
                     [obstacle.get_point()[0] - obstacle.get_radius(), obstacle.get_point()[1], obstacle.get_height() + (Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS)]
                 ]
+                #for new_point in new_attempt_pos_points:
+                #    print("New Attempt Pos Point", str(new_point))
 
                 new_paths = []
                 for new_pos_point in new_attempt_pos_points:
-                    if not self.does_path_intersect_obstacle_3d(obstacle, self.drone.get_point(), new_pos_point) and self.flight_boundary.is_point_in_bounds(new_pos_point):
+                    print(self.flight_boundary.is_point_in_bounds(new_pos_point))
+                    if not self.does_path_intersect_obstacle_2d(obstacle, self.drone.get_point(), new_pos_point) and self.flight_boundary.is_point_in_bounds(new_pos_point):
+                    #if self.flight_boundary.is_point_in_bounds(new_pos_point):
```

Comment:
Remove the commented out code and the print statement.

Reply:
done